cmd/commento/main.go

```diff
@@ -15,23 +16,23 @@ func main() {
 	if err != nil {
 		Die(err)
 	}
-	
+
+	// Load configuration from the environment.
+	// Values in earlier files will take precedence over later values
```

Comment:
you mean "values in **later** files will take precedence" right? Because a setting in `.env.development.local` will be overwritten if the same setting has a different value in `.env`, which mean `.env` takes precedence over `.env.development.local`